### DISCUSSION TITLE
Parallel FEMM

### DISCUSSION THREAD

**Discussion by SebGue** [Eomys/pyleecan#269] [Q&A]
Hello at all, I had some issues with parallel MagFEMM recently, where number of steps < number of workers. Did you test this in the past?

**Comment by BonneelP** [comment]
I had some strange behavior as well but I didn't make the link with the number of step. There is also test_EM_SynRM_FL_001 that don't work any more. Maybe now that we have the handler_list we can update how we handle the creation of the handler for FEMM.
